packages/headless/package.json

```diff
@@ -11,10 +11,13 @@
   "exports": {
     ".": {
       "default": "./src/index.ts"
+    },
+    "./playwright": {
+      "default": "./src/playwright.ts"
     }
   },
   "scripts": {
-    "build": "tsc --noEmit",
+    "build": "vue-tsc --build --force",
```

Comment:
Why is --force needed?

Reply:
tbh I am not sure if it is needed, but it ensures that the tsc compiler is run, even when nothing has changed.
You alse get it if you create a clean vue setup using `npm create vue`. So I assume it's recommended or a Best Practice.

Reply:
Hmm okay :) `--force` always sounds like a "better not to do" but if its the vue default lets keep it for now